core/services/ocr2/plugins/ocr2keeper/custom_telemetry.go

```diff
@@ -0,0 +1,152 @@
+package ocr2keeper
```

Comment:
We can extract telemetry related code into a package `telemetry` within `evm21` and to load it from there, so in future versions we could differentiate telemetry packages by versions

Reply:
there already exists a telemetry package in
`core/services/telemetry`, should i just create a new package name like autotelemetry